src/main/kotlin/org/janelia/saalfeldlab/labels/blocks/CachedLabelBlockLookup.kt

```diff
@@ -0,0 +1,5 @@
+package org.janelia.saalfeldlab.labels.blocks
+
+import net.imglib2.cache.Invalidate
+
+interface CachedLabelBlockLookup : LabelBlockLookup, Invalidate<LabelBlockLookupKey>
```

Comment:
Good for now, maybe in the future, we just will change `LabelBlockLookup` to
```kotlin
interface LabelBlockLookup : Invalidate<LabelBlockLookupKey>
```
or similar.